src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/ControlFlowBuilder.cs

```diff
@@ -291,13 +332,15 @@ internal void CopyCodeAndFixupBranches(BlobBuilder srcBuilder, BlobBuilder dstBu
                         dstBuilder.WriteInt32(branchDistance);
                     }
 
-                    srcOffset += sizeof(byte) + operandSize;
+                    srcOffset += operandSize;
 
                     // next branch:
                     branchIndex++;
                     if (branchIndex == _branches.Count)
                     {
-                        branch = new BranchInfo(int.MaxValue, label: default, opCode: default);
+                        // We have processed all branches. The MaxValue will cause the rest
+                        // of the IL stream to be directly copied to the destination blob.
+                        branch = new BranchInfo(operandOffset: int.MaxValue, default, default, default, default);
```

Comment:
>default [](http://example.com/codeflow?start=77&length=7)

Keep the argument names. It's not clear what default, default, default, default means.

Reply:
Done.